ui/src/components/bounding-box-tool/utils.ts

```diff
@@ -0,0 +1,209 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { PointerEvent, SVGProps } from 'react';
+
+import { Point, RegionOfInterest } from '../shapes/interfaces';
+
+export const getBoundingBoxInRoi = (boundingBox: RegionOfInterest, roi: RegionOfInterest) => {
```

Comment:
I assumed all of these helpers would already come imported from Geti Classic

Reply:
what do you mean?

Reply:
We already have those helpers in Geti Classic. I assumed they’d be exported and reused here